obspy/imaging/scripts/scan.py

```diff
@@ -244,9 +244,12 @@ def main(argv=None):
         matplotlib.use("agg")
     global date2num
```

Comment:
If there is one I can't see it right now. ;)
Anyway, that's some historic thing I guess and not related to this PR.